lib/scripts/sort-translations.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import fs from 'fs'
+import path from 'path'
+
+import { sortObj } from 'jsonabc'
+import { argv } from 'yargs'
+
+const { fix } = argv
+
+const LOCALES = ['cs-CZ', 'en-US', 'es-ES', 'fr-FR', 'sk-SK']
+
+let status = 0
+
+LOCALES.forEach(file => {
+  const filepath = path.join(__dirname, `../src/locales/${file}.json`)
```

Comment:
Petite suggestion parce que file n'est pas vraiment un file ici:

```suggestion
LOCALES.forEach(locale => {
  const file = `${locale}.json`
  const filepath = path.join(__dirname, `../src/locales/${file}`)
```